libs/utils/src/serde_percent.rs

```diff
@@ -0,0 +1,83 @@
+//! A serde::Desierialize type for percentages.
+//!
+//! See [`Value`] for details.
+
+use serde::{Deserialize, Serialize};
+
+/// If the value is not an integer between 0 and 100,
+/// deserialization fails with a descriptive error.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct Value(#[serde(deserialize_with = "deserialize_pct_0_to_100")] u8);
```

Comment:
NIT: Let it be Percent then instead of plain value?

Reply:
Will address, and convert usage site to unqualified use.